src/include/dhandle.h

```diff
@@ -112,4 +112,18 @@ struct __wt_data_handle {
 #define WT_DHANDLE_OPEN 0x80u         /* Handle is open */
                                       /* AUTOMATIC FLAG VALUE GENERATION STOP */
     uint32_t flags;
+
+/* AUTOMATIC FLAG VALUE GENERATION START */
+#define WT_DHANDLE_ASSERT_TS_READ_ALWAYS 0x001u      /* Assert read always checking. */
```

Comment:
I think the new comments are not correct. This one means `assert read timestamps always used`. And `NEVER` means `assert read timestamps never used`.

Reply:
I think it's correct. It's saying that the flag is using `always` checking or `never` checking. It makes more sense if you look at the comments for the other flags.

I imagine it was written like this because the flag generation script won't let us wrap a comment to the next line. So we can't be too verbose here.